solidity/core/contracts/Outbox.sol

```diff
@@ -138,9 +138,9 @@ contract Outbox is IOutbox, Version0, MerkleTreeManager, Common {
      */
     function checkpoint() external override notFailed {
         uint256 count = count();
-        require(count > 0, "!count");
+        require(count > 1, "!count");
```

Comment:
I think a comment here or in the natspec saying that checkpoints of index 0 are disallowed (and why) would be nice for future readers